corsheaders/middleware.py

```diff
@@ -112,9 +112,13 @@ def process_response(self, request, response):
 
         if conf.CORS_MODEL is not None:
             model = apps.get_model(*conf.CORS_MODEL.split('.'))
-            if model.objects.filter(cors=url.netloc).exists():
+            if model.objects.filter(cors__iendswith=url.netloc).exists():
                 response[ACCESS_CONTROL_ALLOW_ORIGIN] = origin
 
+
+        if conf.CORS_ALLOW_CREDENTIALS:
+            response[ACCESS_CONTROL_ALLOW_CREDENTIALS] = 'true'
+
         if (
             not conf.CORS_ORIGIN_ALLOW_ALL and
             not self.origin_found_in_white_lists(origin, url) and
```

Comment:
the below `check_signal` call should be deleted, it's already been checked once in `is_enabled`